### PR TITLE
Tbd based on iana

### PR DESCRIPTION
**Pull request by gmandyam**

### CODE REVIEW

File: cddl/claim-labels.cddl [hunk 1]
```diff
@@ -18,14 +18,14 @@ submods-label          = JC< "submods",    266 >
 ; These are intentionally above 24 so as to not use up
```
Comment (laurencelundblade) [inline cddl/claim-labels.cddl:18]: Should remove this comment since it isn't true any more.

File: cddl/claim-labels.cddl [hunk 2]
```diff
@@ -18,14 +18,14 @@ submods-label          = JC< "submods",    266 >
...
 ; single-byte labels.
 
-uptime-label           = JC< "uptime",     267 >
+uptime-label           = JC< "uptime",     261 >
```
Comment (laurencelundblade) [inline cddl/claim-labels.cddl:21]: Would be good to put these in numeric order.

File: cddl/claim-labels.cddl [hunk 3]
```diff
@@ -18,14 +18,14 @@ submods-label          = JC< "submods",    266 >
...
-measurements-label     = JC< "measurements", 274 >
-measurement-results-label = JC< "measres" , 275 >
-boot-count-label       = JC< "bootcount",  276 >
+intended-use-label     = JC< "intuse",     275 >
+dloas-label            = JC< "dloas",      269 >
+sw-name-label          = JC< "swname",     270 >
+sw-version-label       = JC< "swversion",  271 >
+manifests-label        = JC< "manifests",  272 >
+measurements-label     = JC< "measurements", 273 >
+measurement-results-label = JC< "measres" , 274 >
```
Comment (laurencelundblade) [inline cddl/claim-labels.cddl:29]: Maybe re-indent all the other lines to be the same as this line.

**Review by laurencelundblade — CHANGES_REQUESTED** [review]
Changes to deb.cddl are needed in the comments.

Changes to the Makefile are needed where substitution is done for TBDs.

Basically, search the rendered document and all the document source for "TBD" to make sure you got them all (except for the ones that are for the RFC editor to fix).

Getting close to done with the whole thing :-)

File: Makefile [hunk 4]
```diff
@@ -43,8 +43,8 @@ draft-ietf-rats-eat.md: $(NC_COMMON_CDDL_FRAGS) \
 nc-cddl/%.cddl: cddl/%.cddl
 	mkdir -p nc-cddl
 	sed 's/;.*//' $< | \
-           sed -e 's/267/TBD/;s/268/TBD/;s/269/TBD/;s/270/TBD/;s/271/TBD/' | \
```
Comment (laurencelundblade) [inline Makefile:46]: For my build env, these lines have to be completely removed.

**Review by laurencelundblade — CHANGES_REQUESTED** [review]
There's still a TBD602 in the .md document.  Plus the Makefile doesn't build for me.

File: Makefile (hunk 4 above)
Comment (gmandyam) [reply Makefile:46]: I have commented them out,  but I don't see how the nc-cddl directory gets created by doing so.  I am still debugging other parts of my env to get the Makefile running, but leaving in those lines allowed for the nc-cddl directory to be created.

Comment (laurencelundblade) [reply Makefile:46]: Just remove the two sed lines that do the "TBD" substitutions.

**Review by laurencelundblade — CHANGES_REQUESTED** [review]

**Comment by gmandyam** [comment]
@laurencelundblade 

Makefile now runs on my end and rendered test has no TBD's.  Please re-run on your end, and if there are no further changes required then merge.  Thanks.

**Pull request merged**